File: src/main/java/io/github/et/conopt4j/logger/Level.java
package io.github.et.conopt4j.logger;

public enum Level {
    INFO("Level.INFO"), DEBUG("Level.DEBUG"), FINE("Level.FINE");
    private final String name;
    Level(String s) {
        this.name = s;
    }
    @Override
    public String toString() {
        return name;
    }
}

File: src/main/kotlin/io/github/et/conopt4j/logger/Level.kt
package io.github.et.conopt4j.logger


/**
 * The levels contain Level.INFO, Level.DEBUG and Level.FINE
 *
 * For Level.INFO, you can call: info(), error(), fatal()
 *
 * For Level.DEBUG, you can call: info(), error(), fatal(), debug(), severe(), fine()
 *
 * For Level.FINE, you can call: info(), error(), fatal(), severe(), fine()
 *
 * If you call the functions that does not match the level you have declared, a LevelNotMatchException will be thrown
 *
 * @author Enderman-Teleporting
 */
enum class Level(val level:String) {
    INFO("Level.INFO"), DEBUG("Level.DEBUG"), FINE("Level.FINE")

}